xml.go

```diff
@@ -215,6 +216,37 @@ func (ctrl Controller) WriteTagsStruct(tl TypeList, wr io.Writer) error {
 	return err
 }
 
+// WriteTagComments writes each tag with its description.
+// It is formatted as valid CSV.
+func (ctrl Controller) WriteTagDescriptions(tl TypeList, wr io.Writer) error {
+	out := csv.NewWriter(wr)
+	record := make([]string, 2)
+
+	for _, tag := range ctrl.Tags {
+		record[0] = tag.Name
+		record[1] = strings.TrimSpace(tag.Description.Cdata)
```

Comment:
I know what you have is slightly more efficient, but I think this reads better:
```suggestion
		record := []string {
				tag.Name,
				strings.TrimSpace(tag.Description.Cdata),
		}
```